src/System.Windows.Forms/tests/UnitTests/System/Resources/ResXResourceWriterTests.cs

```diff
@@ -38,4 +47,237 @@ public void TestRoundTrip()
             }
         }
     }
+
+    [Fact]
+    public void ResXResourceWriter_Constructor_FileName_CreatesInstance()
+    {
+        // Arrange
+        string fileName = _ResxFileName;
+
+        // Act
+        using ResXResourceWriter writer = new ResXResourceWriter(fileName);
```

Comment:
seems you missed my other suggestion here. We avoid type name twice in the declarations like this.  This is applicable to all the changes you made in this file.

```suggestion
        using ResXResourceWriter writer = new(fileName);
```

Reply:
Fixed for this instance and other instances in the file